packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.controller.js

```diff
@@ -16,17 +17,62 @@ export default class SoftphoneController {
     this.$translate = $translate;
     this.TucToast = TucToast;
     this.$q = $q;
+    this.showEditFormSoftphoneName = {};
+    this.model = {};
   }
 
   $onInit() {
     this.getMobileOperatingSystem();
     this.MOBILE_OS = MOBILE_OS;
     this.guide = {
-      title: 'mocked_title',
-      url: 'mocked_url',
-      external: 'mocked_external',
-      name: 'mocked_name',
+      title: 'telephony_line_softphone_guide_title',
+      url: GUIDE_URL,
+      external: GUIDE_URL,
+      name: 'telephony_line_softphone_configuration_name',
```

Comment:
```suggestion
      title: 'telephony_line_softphone_configuration_name',
      url: GUIDE_URL,
      external: true,
```
The translate of button "guide" can be put directly in html. External require a boolean